OnCode9/pixel-art
# Updated the favicon

### Description

Changed the favicon to an svg found here:
   - https://www.svgrepo.com/svg/269301/paint-palette-art
It has the [CC0 license](https://www.svgrepo.com/page/licensing/#CC0)

### QA steps

Preview URL: https://update-favicon.pixel-art-8rh.pages.dev

1. Navigate to: https://update-favicon.pixel-art-8rh.pages.dev
2. Verify the favicon displays on the browser tab.
